### DISCUSSION TITLE
certificate of authorization

### DISCUSSION THREAD

**Discussion by tvelliquette** [ess-dive-community/essdive-uas#11] [General]
Would it be important to capture the certificate of authorization for the flights? I'm looking over some metadata that includes the certificate along with a visual observer when there is one. Did I miss where this is recorded? Or is this information saved elsewhere with the researcher or the project like any land access and sample collection permits?

**Comment by regnans** [comment]
Yes, we agree this is useful to capture. Inclusion of the COA details was added to the latest version. It is part of the Campaign metadata, and is included in the "Permits" variable. Another related new addition is of the name and contact details for the Pilot In Command (PIC), but we haven't added observer.
